fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -749,7 +749,9 @@ private List<String> checkAndExtractPartitionColForRange(Expr expr) {
         if (expr instanceof FunctionCallExpr) {
             FunctionCallExpr functionCallExpr = (FunctionCallExpr) expr;
             String functionName = functionCallExpr.getFnName().getFunction();
-            if (FunctionSet.SUBSTR.equals(functionName) || FunctionSet.SUBSTRING.equals(functionName)) {
+            if (FunctionSet.SUBSTR.equals(functionName)
+                    || FunctionSet.SUBSTRING.equals(functionName)
+                    || FunctionSet.FROM_UNIXTIME.equals(functionName)) {
```

Comment:
better to be removed to Analyzer

Reply:
This is done in the parse stage to check in advance.

Reply:
Be better to use a `Set` or `Map` to refactor it later.